src/Features/Core/Portable/InheritanceMargin/AbstractInheritanceMarginService_Helpers.cs

```diff
@@ -751,5 +697,16 @@ private static ImmutableArray<InheritanceTargetItem> GetNonNullTargetItems(Immut
 
             return builder.ToImmutable();
         }
+
+        internal static bool IsNavigableSymbol(ISymbol symbol)
+        {
+            var locations = symbol.Locations;
+            if (locations.Length == 1)
+            {
+                var location = locations[0];
+                return location.IsInMetadata || (location.IsInSource && location.IsVisibleSourceLocation());
+            }
+            return !locations.IsEmpty;
```

Comment:
not sure i get the logic here.  it's navigable if it's either got one location that hav esome property, or it has 2 or more locations total?